.github/README.md

```diff
@@ -10,7 +10,7 @@ Oppia is written using Python and AngularJS, and is built on top of Google App E
   * [User Documentation](https://oppia.github.io/)
   * [Contributors' wiki](https://github.com/oppia/oppia/wiki)
   * [Developer mailing list](http://groups.google.com/group/oppia-dev)
-  * [File an issue](https://github.com/oppia/oppia/issues/new?title=Describe%20your%20feature%20request%20or%20bug%20report%20succinctly&body=If%20you%27d%20like%20to%20propose%20a%20feature,%20describe%20what%20you%27d%20like%20to%20see.%20Mock%20ups%20would%20be%20great!%0A%0AIf%20you%27re%20reporting%20a%20bug,%20please%20be%20sure%20to%20include%20the%20expected%20behaviour,%20the%20observed%20behaviour,%20and%20steps%20to%20reproduce%20the%20problem.%20Console%20copy-pastes%20and%20any%20background%20on%20the%20environment%20would%20also%20be%20helpful.%0A%0AThanks!)
+  * [File an issue](https://github.com/DubeySandeep/oppia/issues/new/choose)
```

Comment:
I think your link is wrong, unless you want them to report it on your own issue tracker :D

Reply:
Oops! sorry, I was checking in my own repo and did a copy paste :p

Reply:
Done!